source/sessions/tests/unified/snapshot-sessions-not-supported-server-error.yml

```diff
@@ -3,24 +3,21 @@ description: snapshot-sessions-not-supported-server-error
 schemaVersion: "1.0"
 
 runOnRequirements:
-  - minServerVersion: "3.6"
-    maxServerVersion: "4.4.99"
-  - minServerVersion: "3.6"
+  - minServerVersion: "5.0"
     topologies: [ single ]
 
 createEntities:
   - client:
       id: &client0 client0
       observeEvents: [ commandStartedEvent, commandFailedEvent ]
-      ignoreCommandMonitoringEvents: [ findAndModify, insert, update ]
   - database:
-      id: &database0 database0
+      id: &database0Name database0
       client: *client0
-      databaseName: &database0Name database0
+      databaseName: *database0Name
   - collection:
-      id: &collection0 collection0
-      database: *database0
-      collectionName: &collection0Name collection0
+      id: &collection0Name collection0
+      database: *database0Name
+      collectionName: *collection0Name
```

Comment:
I'm not sure I understand the point of this. Previously, `database0` pointed to the entity ID (to be used in operations), while `database0Name` pointed to the name of the database (which doesn't necessarily have to match). However, the entity ID was never referenced in the tests below, instead using `database0` as hard-coded identifier. IMO, these entities should remain the same (`database0` for the entity, `database0Name` for the name of the database backing the entity), but the tests below should use references instead of hardcoding entity IDs.

As a quick resolution to not stall the downstream work based on this, I'd be fine with undoing this change and filing a follow-up ticket to ensure we're using references when using entities.

Reply:
&database0Name and &database0 are exactly the same (they're both "database0"). All I did was remove the redundant label. Is that a problem?

Reply:
I'm going to merge this with the changes I already made. I found it confusing to have multiple labels for the same string especially when they're barely used. If you'd like to change it back or standardize on one way or the other that's fine by me. Perhaps there's even a tool to automatically "condense" yaml files.

Reply:
Yeah, no need to change it here, just wanted to point it out. Use of these are spotty at best across our spec tests (see this file not using the entity reference and instead hardcoding the name), so no need to enforce consistency when there isn't any to begin with.